index.html

```diff
@@ -1,36 +1,34 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta name="author" content="Keri West" />
-    <meta name="description" content="Keri West's website is a digital portfolio to display his coding work and a little about himself."/>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="author" content="Keri West">
+    <meta name="description" content="Keri West's website is a digital portfolio to display his coding work and a little about himself.">
 
     <title>Keri West's Website</title>
 
-    <link rel="stylesheet" type="text/css" href="css/index.css" />
-    <link href="https://fonts.googleapis.com/css2?family=Belanosima:wght@600&family=Roboto+Mono:wght@300;400&display=swap" rel="stylesheet"/>
+    <link rel="stylesheet" type="text/css" href="css/index.css">
+    <link href="https://fonts.googleapis.com/css2?family=Belanosima:wght@600&family=Roboto+Mono:wght@300;400&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
   </head>
 
   <body id="top">
     <div>
+      
       <nav>
-        <a class="nav-icon" href="#top"><img src="img/icons/KWEST_Logo_v01.svg" alt="kwest site icon"/></a>
+        <a class="nav-icon" href="#top"><img src="img/icons/KWEST_Logo_v01.svg" alt="kwest site icon"></a>
         <ul class="nav-menu">
-          <a class="nav-item-nav-icon" href="#top"><img src="img/icons/KWEST_Logo_v01.svg" alt="kwest site icon"/></a>
+          <a class="nav-item-nav-icon" href="#top"><img src="img/icons/KWEST_Logo_v01.svg" alt="kwest site icon"></a>
           <li class="nav-item">
-            <a href="#about" class="nav-link">About me</a>
-          </li>
+            <a href="#about" class="nav-link">About me</a></li> 
```

Comment:
```suggestion
            <a href="#about" class="nav-link">About me</a>
          </li> 
```
Slight preference to keep the closing tag on a line of its own, if it spans multiple lines.